### DISCUSSION TITLE
Elettrovalvole acqua e GPL

### DISCUSSION THREAD

**Discussion by Jocman** [indomus/forum#2037] [GENERALE - Discussioni generiche]
E' un pò che dovrei farlo a casa mia, e vorrei approfittare della ristruturazione di un piccolo trilocale di fu mia nonna per....sperimentare.
Ci staccheremo dal gas cittadino (la casa verrà utilizzata saltuariamente), ma ci era stato suggerito di installare una caldaia a GPL (un domani convertibile in metano e quindi alla rete cittadina) collegata ad una bombola di GPL da 25kg.
L'idea era quella di installare una elettrovalvola (e manometro, così da avere info sul carico) domotizzabili, in modo da poter accendere (ad es un paio di ore prima di arrivare) da remoto la caldaia e trovare la casa un pò in temperatura.
Elettrovalvole in commercio ne esistono di tutti i gusti, ma  il mio dubbio era se prenderle NC o NA (ma le prenderei comunque a riarmo manuale).
Per logica, tenderei a una tipologia NC: se per un qualche motivo manca la corrente, tutti i flussi (acqua e gas) verrebbero interrotti.
La scelta NC la farei anche alla luce di una disavventura che mi è capitata qualche anno fa a casa: rottura di tubazione nel seminterrato, si allaga il seminterrato (sembrava una piscina indoor.....) e salta la corrente perchè l'acqua è arrivata alle prese. Con una elettrovalvola NC domotizzata (tralasciando il fatto che un sensore acqua avrebbe chiuso l'elettrovalvola impedendo il caos  - con relativi danni - che invece è successo) lo scatto del contatore avrebbe comportato la chiusura della valvola. Certo in caso interruzione per guasto di rete rimarrei senza acqua e gas, ma a questo rimedierei con il riarmo manuale.

Consigli, opinion, suggerimenti?

**Comment by KC-inDomus** [comment]
scegli elettrovalvole certificate [atex](https://it.wikipedia.org/wiki/ATEX)

**Comment by Jocman** [comment]
Ottimo, grazie mille del suggerimento
